changelogs/head.asciidoc

```diff
@@ -9,6 +9,7 @@ https://github.com/elastic/apm-server/compare/6.5\...6.x[View commits]
 - Set some configuration defaults (setup.template.settings.index.*, logging.metrics.enabled) in code {pull}1494[1494].
 - Add `span.sync` property to intake json spec and index field in ES. {pull}1548[1548].
 - Make `service.framework` properties optional and nullable {pull}1546[1546].
+- Add `transaction.sampled` to errors {pull}1662[1662].
```

Comment:
i have a question, is correct here to refer to the original PR or should be the one of the backport?

Reply:
I think we always refer to the original PR.

Reply:
yes, agreed on original pr